src/main/java/mate/academy/Main.java

```diff
@@ -6,18 +6,21 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
+            OrderService orderService = new OrderServiceImpl();
+            orderService.completeOrder(user.getUserId());
         } catch (AuthenticationException e) {
-            e.printStackTrace();
-            return;
```

Comment:
Be careful, by deleting return you changed the actual logic of application, which should not be done in this task